design/sketch.webidl

```diff
@@ -815,10 +815,20 @@ dictionary GPURequestAdapterOptions {
 };
 
 [Exposed=Window]
```

Comment:
Shouldn't moving `gpu` to `navigator` just require changing `Exposed=Window` to `Exposed=Navigator`?

Reply:
Hm, I copied this from WebUSB and Web Bluetooth, but I don't see anything in Web IDL that would prevent it.